src/main/java/nl/ordina/jobcrawler/service/MatchSkillsService.java

```diff
@@ -27,14 +27,16 @@
 public class MatchSkillsService {
```

Comment:
I think a better name would be SkillMatcherService

Reply:
Ok, will adapt